app/helpers/if-filter.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function ifFilter(params) {
+    if ((typeof params[1] === 'undefined') || ((params[0].toLowerCase()).indexOf(params[1].toLowerCase()) !== -1)){
```

Comment:
Can you please add a comment describing the purpose of this helper?

We use YUIDoc format- here is a sample:
https://github.com/abought/ember-osf/blob/9264fb5e8cdc52f30436bd9668f8a2905a0389a4/addon/helpers/elem-id.js#L9-L9